### DISCUSSION TITLE
How do I install the latest developer version?

### DISCUSSION THREAD

**Discussion by Lyumih** [mozilla/pdf.js#17286] [Q&A] [answered]
How do I install the latest developer version?
Bug https://github.com/mozilla/pdf.js/issues/16839 it was fixed earlier, but the release version has not yet been released. 

I have 2 questions.
1. How to install version 4.0.240 above release 4.0.189?
2. This bug https://github.com/mozilla/pdf.js/issues/16839 will it be fixed in version 3 and 4 or only in version 4? We are now on version 3, version 4 breaks the application and we will have to fix it.

Thanks!

**Comment by Snuffleupagus** [comment, marked as answer]
> How do I install the latest developer version?

Sorry, but it's *really not* a good idea (nor officially supported) to use that in production environments since doing so *may* cause problems (now or later).

> How to install version 4.0.240 above release 4.0.189?

Please don't do that, and wait for the next official release instead. The next release, from the `4`-branch, is planned for the end of the month.

> [...] will it be fixed in version 3 and 4 or only in version 4?

Unfortunately no more development, or releases, will happen for the `3`-branch at this point.
(In the PDF.js project we almost never back-port patches to previous releases, given the amount of work involved in doing so.)

**Reply by Lyumih** [reply]
Thanks for answers!
